python/s3-sns-sqs-lambda-chain/requirements.txt

```
@@ -0,0 +1,2 @@
+aws-cdk-lib==2.12.0
+constructs>=10.0.0,<11.0.0
```

```suggestion
aws-cdk-lib>=2.12.0
constructs>=10.0.0
```
Lets keep this open to the latest CDK. This way we can use CI to identify when if it fails with a CDK update.